### PR TITLE
VideoCommon: enable vulkan in windows solution, update CMake vulkan option to match opengl

### PR DESCRIPTION
**Pull request by iwubcode**

### CODE REVIEW

**Comment by iwubcode** [comment]
An alternative to #9189

File: Source/Core/VideoCommon/VideoCommon.vcxproj [hunk 1]
```diff
@@ -15,9 +15,9 @@
     <Import Project="$(VSPropsDir)PCHUse.props" />
   </ImportGroup>
   <PropertyGroup Label="UserMacros" />
-  <ItemDefinitionGroup>
+  <ItemDefinitionGroup Condition="'$(Platform)'=='x64'">
```
Comment (leoetlino) [inline Source/Core/VideoCommon/VideoCommon.vcxproj:18]: The platform condition should probably only apply to HAS_OPENGL

**Review by leoetlino — APPROVED** [review]
LGTM

**Comment by JMC47** [comment]
Fixes Vulkan not showing up in windows

File: CMakeLists.txt [hunk 2]
```diff
@@ -560,6 +560,11 @@ if(ENABLE_VULKAN)
   add_definitions(-DUSE_VULKAN)
 endif()
 
+if(NOT WIN32 OR (WIN32 AND NOT (CMAKE_SYSTEM_PROCESSOR STREQUAL "aarch64")))
```
Comment (JosJuice) [inline CMakeLists.txt:563]: There's no reason to check `WIN32` to the right of `OR`.

**Review by JosJuice — APPROVED** [review]

**Pull request merged**